Fixing for Clipboard-Agent Coverity issues

alavalas1 opened the pull request:
Fixing for Clipboard-Agent Coverity issues

Coverity scan reported
exn_spec_violation An exception of type std::length_error

Test done: Boot and Functional test passed with engineering build.

Tracked-On: OAM-112451

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed Engineering Build for this issue, build is SUCCESS.To merge the changes please click on APPROVE FOR MERGE button in Android CI WEB

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed Engineering Build for this issue, build is SUCCESS.To merge the changes please click on APPROVE FOR MERGE button in Android CI WEB

alavalas1 committed:
Fixing for Clipboard-Agent Coverity issues

Coverity scan reported
exn_spec_violation An exception of type std::length_error

Test done: Boot and Functional test passed with engineering build.

Tracked-On: OAM-112451
Signed-off-by: Reddy, Alavala Srinivasa <alavala.srinivasa.reddy@intel.com>

sysopenci commented:
Improper Commit Message
Tracked on not found in commit message,
    make sure Tracked-On: Jira-ticket  is present.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has completed Engineering Build for this issue, build is FAILURE. Please check the linked Tracked-On issue/Android CI Web  for more details

vinaykompella reviewed (commented):
LGTM

sysopenci commented:
Android CI has completed Engineering Build for this issue, build is SUCCESS.To merge the changes please click on APPROVE FOR MERGE button in Android CI WEB

shyjumon-n reviewed (approved):
LGTM

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon-preintegration/builds/5290

Done by cdsart <cdsart@intel.com>

sysopenci reviewed (commented):
PR integrated into 'premerge' builder



More details, logs, and binaries: /absp/builders/celadon-premerge/builds/3077

Done by cdsart <cdsart@intel.com>

sysopenci reviewed (commented):
cactus finished building and testing your changes 

It was requested for merge by cdsart <cdsart@intel.com> 

so we merge it. Thanks for your contribution.

more details, logs, and binaries: /absp/builders/celadon-mergerequest/builds/3430
